### ISSUE TITLE
rdpwrap.ini for (1809) 17763.503 - Win10 X64

### ISSUE THREAD

**Issue by SteepNet** [stascorp/rdpwrap#783]
New update windows KB4494441& new version 17763.503
File correction needed rdpwrap.ini 
Can someone help?

**Comment by kuliss** [comment]
Hello,

You need to remove update KB4499167 and then use http://download.microsoft.com/download/f/2/2/f22d5fdb-59cd-4275-8c95-1be17bf70b21/wushowhide.diagcab to block (hide) this update.

**Comment by stvkodra** [comment]
Edit your current ini by adding a blank line and paste the following. Confirmed working.

[10.0.17763.503]
LocalOnlyPatch.x64=1
LocalOnlyOffset.x64=77A41
LocalOnlyCode.x64=jmpshort
SingleUserPatch.x64=1
SingleUserOffset.x64=3E520
SingleUserCode.x64=Zero
DefPolicyPatch.x64=1
DefPolicyOffset.x64=18025
DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
SLInitHook.x64=1
SLInitOffset.x64=1ACDC
SLInitFunc.x64=New_CSLQuery_Initialize

bInitialized.x64      =ECAB0
bServerSku.x64        =ECAB4
lMaxUserSessions.x64  =ECAB8
bAppServerAllowed.x64 =ECAC0
bRemoteConnAllowed.x64=ECAC4
bMultimonAllowed.x64  =ECAC8
ulMaxDebugSessions.x64=ECACC
bFUSEnabled.x64       =ECAD0

**Comment by SteepNet** [comment]
not work =(

**Comment by wyxchari** [comment]
> 
> 
> New update windows KB4494441& new version 17763.503
> File correction needed rdpwrap.ini
> Can someone help?

Windows version 17763.503 of May 14, 2019 uses the termsrv.dll file, version 17763.475. The file with the latest updates has not been modified.

**Comment by NinjaInc** [comment]
Works! Fixed my issue and can connect again!

**Comment by jonathan19091** [comment]
alguien ya lo puedo reparar? 
can I already repair it?

lo único que no puedo quitar el es not listening
the only thing I can not remove is not listening

**Comment by SBRK** [comment]
I might be stupid, but where should I put/what should I do with the rdpwrap.ini file? I don't have an ini file anywhere to replace. If I just put it with the rest of the files and re-run install.bat or update.bat, I still get "not supported"

**Comment by jonathan19091** [comment]
Hola @SBRK  este se coloca en C:\Program Files\RDP Wrapper
en ese lugar se pone el archivo .ini

**Comment by SteepNet** [comment]
problem solved

**Comment by zcq0730** [comment]
> Works! Fixed my issue and can connect again!

Could you share the rdpwrap.ini file?  ( for 17763.503)  Thanks a lot.

**Comment by SBRK** [comment]
I didn't use the msi installer (because Windows Defender) and instead use the zip version. Is the ini file usable in this case ?

**Comment by zcq0730** [comment]
Hey, Yes, it works. I used the ZIP file as well.Check the instruction video, very useful, key thing is to run RDPWinst.exe after updated ini file.Good luck.Von meinem Samsung Galaxy Smartphone gesendet.
-------- Ursprüngliche Nachricht --------Von: Benjamin MICHEL <notifications@github.com> Datum: 19.05.19  12:56  (GMT+01:00) An: stascorp/rdpwrap <rdpwrap@noreply.github.com> Cc: zcq0730 <zcq0730@gmail.com>, Comment <comment@noreply.github.com> Betreff: Re: [stascorp/rdpwrap] rdpwrap.ini for (1809) 17763.503 - Win10 X64 (#783) I didn't use the msi installer (because Windows Defender) and instead use the zip version. Is the ini file usable in this case ?

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or mute the thread.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/stascorp/rdpwrap/issues/783?email_source=notifications\u0026email_token=AJTS632YKFLLX3FYX2H4HQLPWEW6BA5CNFSM4HNA54W2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVW7VSQ#issuecomment-493746890",
"url": "https://github.com/stascorp/rdpwrap/issues/783?email_source=notifications\u0026email_token=AJTS632YKFLLX3FYX2H4HQLPWEW6BA5CNFSM4HNA54W2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGODVW7VSQ#issuecomment-493746890",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by DarkLegacy24** [comment]
This did not work for me does someone have a working ini file???

**Comment by zcq0730** [comment]
This cost me quite some time to figure out. Here is the solution worked for me.
My system windows10 pro,  17763.503.
Check the instruction video, very useful, key thing is to run RDPWinst.exe after updated ini file. 
Finding a working ini file wasn´t the problem.  Check the following comments if needed.
Good luck!
https://www.youtube.com/watch?v=W9BpbEt1yJw

https://github.com/stascorp/rdpwrap/issues/759#issuecomment-493596622
RDPWInst.exe    | RDP Wrapper Library installer/uninstaller
https://github.com/stascorp/rdpwrap/issues/744#issuecomment-493586640
https://github.com/stascorp/rdpwrap/issues/744#issuecomment-493816641
https://github.com/stascorp/rdpwrap/issues/783#issuecomment-492662982

> This did not work for me does someone have a working ini file???

**Comment by bootablearg** [comment]
> Edit your current ini by adding a blank line and paste the following. Confirmed working.
> 
> [10.0.17763.503]
> LocalOnlyPatch.x64=1
> LocalOnlyOffset.x64=77A41
> LocalOnlyCode.x64=jmpshort
> SingleUserPatch.x64=1
> SingleUserOffset.x64=3E520
> SingleUserCode.x64=Zero
> DefPolicyPatch.x64=1
> DefPolicyOffset.x64=18025
> DefPolicyCode.x64=CDefPolicy_Query_eax_rcx
> SLInitHook.x64=1
> SLInitOffset.x64=1ACDC
> SLInitFunc.x64=New_CSLQuery_Initialize
> 
> bInitialized.x64 =ECAB0
> bServerSku.x64 =ECAB4
> lMaxUserSessions.x64 =ECAB8
> bAppServerAllowed.x64 =ECAC0
> bRemoteConnAllowed.x64=ECAC4
> bMultimonAllowed.x64 =ECAC8
> ulMaxDebugSessions.x64=ECACC
> bFUSEnabled.x64 =ECAD0

Do not works for me.

Some Idea ?

**Comment by bootablearg** [comment]
> problem solved

How you fix it ?

**Comment by Vegz78** [comment]
Great program and great support here on the github forum whenever W10 updates, allthough maybe a bit messy and hard to follow...

Any way to update RDPWrapper with needed updates to rdpwrap.ini through RDPWInst.exe, or otherwise keep oneself updated on changes/updates?

Understanding and implementing needed changes to rdpwrap.ini myself seems a bit hard to grasp.

Br,

Vegard

**Comment by ironbelly** [comment]
Running 17763.503 from updates installed 05-16 and none of the above solutions work, still keeps on asking to boot out other users before allowing concurrent RDP sessions to login
